src/App.jsx

```diff
@@ -14,45 +17,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_TYPE_LENG = 'leng';
+const SORT_TYPE_ALPH = 'alph';
+
+const updateGoods = (goods, { sortType, isReverse }) => {
+  if (sortType === '' && !isReverse) {
+    return goods;
+  }
+
+  const udatedGoods = [...goods];
+
+  if (sortType === SORT_TYPE_LENG || sortType === SORT_TYPE_ALPH) {
+    udatedGoods.sort((good1, good2) => {
```

Comment:
let's use 'a' and 'b' instead of good1 and good2, using numbers in variable names is not the best practice